plonk/src/transcript/mod.rs

```diff
@@ -67,6 +67,14 @@ pub trait PlonkTranscript<F> {
             vk.num_inputs.to_le_bytes().as_ref(),
         )?;
 
+        // include [x]_2 G2 point from SRS
+        // all G1 points from SRS are implicit reflected in committed polys
+        <Self as PlonkTranscript<F>>::append_message(
+            self,
+            b"SRS G2 element",
```

Comment:
There is a [TODO](https://github.com/EspressoSystems/jellyfish/blob/b273fc2cb607063aef52e762b2b3452c8ee043fa/pcs/src/univariate_kzg/srs.rs#L22), about removing `beta_h`. 
So should not we use `vk.open_key.powers_of_h[0]` instead?

Reply:
it should be `[1]`, changed in [30c463a](https://github.com/EspressoSystems/jellyfish/pull/647/commits/30c463a06801973d6280c253710201d06d1560d5)

Reply:
Ah right, thanks.